Backend/filesystem/service.go

```diff
@@ -92,3 +92,31 @@ func deleteEntity(pool database.Pool, entityID int) error {
 	}
 	return nil
 }
+
+// renameEntity changes the logical name of a given object in the filesystem
+func renameEntity(pool database.Pool, entityID int, newName string) error {
+	_, err := pool.GetConn().Exec(context.Background(),
+		"UPDATE filesystem SET logicalname = ($1) WHERE entityid = ($2)", newName, entityID)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// // getFilesystemInfo returns information regarding a specific file system entity
+// func getEntityChildren(pool database.Pool, docID int) (EntityInfo, error) {
+// 	entity := EntityInfo{}
+// 	children := pgtype.Hstore{}
+
+// 	err := pool.GetConn().QueryRow(context.Background(), "SELECT EntityID FROM filesystem WHERE Parent = $1",
```

Comment:
this isnt what `getEntityChildren` is meant to do, I explained it to nian on discord, it should be there

Reply:
this is just the getInfo function copy pastad - i stopped when i asked you about the db, will be different for actual implementation

Reply:
aight cool